browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -144,15 +154,15 @@ void BraveToolbarView::Layout() {
   } else {
     home_->SetVisible(false);
   }
-
-  // position Brave's BookmarkButton
-  if (bookmark_ && bookmark_->visible()) {
-    next_element_x += element_padding;
-    bookmark_->SetBounds(next_element_x, toolbar_button_y, bookmark_->GetPreferredSize().width(), toolbar_button_height);
-    next_element_x = bookmark_->bounds().right();
-  }
-  next_element_x += GetLayoutConstant(TOOLBAR_STANDARD_SPACING);
-
+  next_element_x += element_padding;
```

Comment:
we now always want `element_padding` distance between all items, even LocationBar, instead of `GetLayoutConstant(TOOLBAR_STANDARD_SPACING)` for the spacing to the left of LocationBar